R/DAISIE_IC.R

```diff
@@ -30,7 +30,7 @@ DAISIE_IC <- function(datalist,
      tol = c(1e-04, 1e-05, 1e-07),
      maxiter = 1000 * round((1.25) ^ length(idparsopt)),
      methode = "lsodes",
-     optimmethod = "subplex"
+     optimmethod = "simplex"
```

Comment:
Is this supposed to be the default optimizer? If so, then we need to change the documentation. If not, then we need to change the documentation in the functions where `simplex` is the optimizer, because right now the docs all mention that `subplex` is always used by default.

Reply:
Fixed in #179